README.md

```diff
@@ -14,9 +14,9 @@ Geyser is an open collaboration project by [CubeCraft Games](https://cubecraft.n
 
 ## What is Geyser?
 Geyser is a proxy, bridging the gap between Minecraft: Bedrock Edition and Minecraft: Java Edition servers.
-The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
+This project's ultimate goal is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
```

Comment:
Seems silly.

Reply:
I wonder if it seems silly, im not sure perhaps I need another confirmation

Reply:
It's worse haha

Reply:
How is it silly/worse lol. There's no difference and gets to the point quicker

Reply:
yeah original sounds better

Reply:
```suggestion
The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
```